setup/src/Magento/Setup/Console/Command/RollbackCommand.php

```diff
@@ -127,7 +127,13 @@ protected function execute(InputInterface $input, OutputInterface $output)
             if (!$helper->ask($input, $output, $question) && $input->isInteractive()) {
                 return \Magento\Framework\Console\Cli::RETURN_FAILURE;
             }
-            $this->doRollback($input, $output);
+            $questionKeep = new ConfirmationQuestion(
+                '<info>you want to keep the backups?[y/N]<info>',
```

Comment:
How about "Do you want to keep the backup files?" as the message here.

Reply:
Fixed typo